README.md

```diff
@@ -2,12 +2,18 @@
 
 work in progress [prettier plugin](https://github.com/prettier/prettier/issues/4180) for [solidity](https://github.com/ethereum/solidity)
 
-## Use
+## Resources
+
+- https://github.com/mattiaerre/prettier-plugin-solidity/issues/2
+
+## Test
 
 ```bash
-node_modules/prettier/bin-prettier.js --write **/*.sol --plugin=./node_modules/prettier-plugin-solidity
+node_modules/prettier/bin-prettier.js --write **/*.sol --plugin=./src
```

Comment:
This should be `node_modules/.bin/prettier` — the path to the actual CLI could change at any time.

Reply:
of course, thanks for that!